src/controller.cpp

```diff
@@ -399,8 +399,9 @@ Controller::UpdateResult Controller::update(
     ROS_WARN("All three gains (Kp, Ki, Kd) should have the same sign for stability.");
   }
 
-  controller_state_.error_lat.push(error.getOrigin().y());
-  controller_state_.error_ang.push(angles::normalize_angle(tf2::getYaw(error.getRotation())));
+  auto error_lat_filtered = controller_state_.error_lat.filter(error.getOrigin().y(), dt.toSec());
+  auto error_ang_filtered = controller_state_.error_ang.filter(
```

Comment:
There is a bracket mismatch here.
Basically this statement now does:
```c++
auto error_ang_filtered = dt.toSec()
```

Reply:
How can this compile, dt is a mandatory argument for filter?

Reply:
Sorry, I picked the wrong line :slightly_smiling_face: